litmus-portal/graphql-server/pkg/database/mongodb/workflow/schema.go

```diff
@@ -21,20 +25,98 @@ type ChaosWorkFlowInput struct {
 	CreatedAt           string              `bson:"created_at"`
 	ProjectID           string              `bson:"project_id"`
 	ClusterID           string              `bson:"cluster_id"`
+	ClusterName         string              `bson:"cluster_name"`
+	ClusterType         string              `bson:"cluster_type"`
 	WorkflowRuns        []*ChaosWorkflowRun `bson:"workflow_runs"`
 	IsRemoved           bool                `bson:"isRemoved"`
 }
 
-// ChaosWorkflowRun contains the required fields to be stored in the database for a workflow run
-type ChaosWorkflowRun struct {
-	WorkflowRunID string `bson:"workflow_run_id"`
-	LastUpdated   string `bson:"last_updated"`
-	ExecutionData string `bson:"execution_data"`
-	Completed     bool   `bson:"completed"`
-}
-
 // WeightagesInput contains the required fields to be stored in the database for a weightages input
 type WeightagesInput struct {
 	ExperimentName string `bson:"experiment_name"`
 	Weightage      int    `bson:"weightage"`
 }
+
+// ChaosWorkflowRun contains the required fields to be stored in the database for a workflow run
+type ChaosWorkflowRun struct {
+	WorkflowRunID     string   `bson:"workflow_run_id"`
+	LastUpdated       string   `bson:"last_updated"`
+	Phase             string   `bson:"phase"`
+	ResiliencyScore   *float64 `bson:"resiliency_score,string,omitempty"`
+	ExperimentsPassed *int     `bson:"experiments_passed,string,omitempty"`
+	TotalExperiments  *int     `bson:"total_experiments,string,omitempty"`
+	ExecutionData     string   `bson:"execution_data"`
+	Completed         bool     `bson:"completed"`
+}
+
+type ExecutionData struct {
+	WorkflowID        string          `json:"-"`
```

Comment:
```suggestion
	WorkflowID        string          `bson:"-"`
```

please change this to other places

Reply:
these are used for parsing the ExecutionData...we are storing it the way it was before as a string...hence json. we moved it here as in future PRs this struct will be shared. It was previously inside operations.go. @rajdas98

Reply:
If its specific for just the operations, you can move it to workflow types @arkajyotiMukherjee

Reply:
Yes if it isn't related to database it shouldn't be in the db pkg

Reply:
Also this is not updated, there have been changes to subscriber and this needs to be updated too